Makefile

```
@@ -101,7 +101,7 @@ check-tidy: generate
 release:
 	@(VERSIONED_FILES="version.go \
 	npm-packages/cadence-parser/package.json" \
-	./bump-version.sh $(bump))
+	bash ./bump-version.sh $(bump))
```

```suggestion
	sh ./bump-version.sh $(bump))
```

for better portability. `bump-version.sh` uses POSIX `sh` anyway

---

Actually, I meant to use bash here but had forgotten to update the POSIX. The version-bump command (string comparison) is not platform-independent in shell, hence needs bash.